src/partners/index.njk

```diff
@@ -32,7 +32,7 @@ description: FlowFuse partners with leading cloud, hardware, technology and cons
                     <a href="/partners/ctrlx/">{% image "../images/partners/logo-rexroth.png", "Rexroth logo", [150] %}</a>
                 </li>
                 <li class="w-[150px] pb-6 md:pb-0 hover:transition-opacity hover:opacity-75 transition ease-in duration-300">
-                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "Rexroth NCD.io", [150] %}</a>
+                    <a href="/partners/ctrlx/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```

Comment:
@Yndira-E Please look at the href too:

```suggestion
                    <a href="https://ncd.io/">{% image "../images/partners/logo-ncd-io.png", "NCD.io logo", [150] %}</a>
```

Reply:
Thanks @ZJvandeWeg, and sorry! Seems like I'm trying to go too fast